src/backend/db/users.js

```diff
@@ -9,10 +9,9 @@ import { formatDate } from "../utils/authUtils";
 export const users = [
   {
     _id: uuid(),
-    firstName: "Adarsh",
-    lastName: "Balika",
-    email: "adarshbalika@gmail.com",
-    password: "adarshBalika123",
+    username: "Roronoa Zoro",
```

Comment:
Better if you keep test login as something simple like guest.

Reply:
🤣